web/alerting/status_gluster_afr.rst

```diff
@@ -21,8 +23,107 @@ Based on:
 Setup
 =====
 
-#. You need a gluster volume.
-   TODO: list of volumes from qe configurations which uses AFR translator
+#. You need a gluster arbiter volume.
+   (distributed, distributed-replicate) volumes uses AFR translator
+   to this categories falls also arbiter - distributed-replicate volume
+
+Test Steps
+==========
+
+.. test_action::
+    :step:
+        List *replica sets* - *subvolumes* of present arbiter volume
+
+        .. code-block:: console
+
+            # gluster volume info
+    :result:
+        all bricks are printed
+
+.. test_action::
+    :step:
+        Choose some replica set (subvolume) and stop *glusterd* service
+        on machine, where arbiter brick from the subvolume is located.
+        Stop the *glusterd* service on the machine where one of the
+        subvolume data bricks is located too.
+
+        .. code-block:: console
+
+            # systemctl stop glusterd
+    :result:
+        A *glusterd* service is stopped on both machines.
+        **Tendrl** generates an (warning) alert for this event.
```

Comment:
Which one?

Reply:
For that there's another test, that's just a note that it will happen. No need to verify this.
However you are right, there should be plural alerts/events not singular alert/event.